app/presenters/export/csv/incoming_trust_presenter_module.rb

```diff
@@ -23,6 +23,10 @@ def incoming_trust_name
     @project.incoming_trust.name
   end
 
+  def incoming_trust_main_contact
+    @project.incoming_trust_main_contact_id
```

Comment:
This is only going to return an ID, which isn't very useful for the consumer. Can you get the `User` record from this ID, and return the name or email address (I'm going to assume email address but check with Steve)